content/nginx-one/connect-instances/connect-using-squid.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Prepare - Set up Squid as a proxy
+toc: true
+weight: 250
+nd-docs: DOCS-000
+---
+
+NGINX Agent can be configured to connect to NGINX One using a [Squid proxy](https://www.squid-cache.org/). This is useful in environments where direct internet access is restricted or monitored.
+
+## Before you start
+
+Ensure you have the following:
+
+- [Squid proxy server set up and running](https://wiki.squid-cache.org/SquidFaq/InstallingSquid)
+- [NGINX Agent is installed]({{< ref "nginx-one/agent/install-upgrade/" >}})
+- Access to the NGINX One console
+
+## Install Squid
+
+Follow the instructions in the [Squid website](https://wiki.squid-cache.org/SquidFaq/BinaryPackages) to install Squid on your server.
+
+
+## Configure Squid
+
+Follow the steps below to configure Squid with basic authentication.
+
+1. Open the Squid configuration file with your favorite text editor (you might need superuser privileges):
```

Comment:
In general I'm not a fan of this nested-list pattern: I find it a poor substitute to avoid headings. Tends to create formatting problems with multi-line instructions and code blocks.

I'd trust the fact the document is being read linearly from top to bottom and just have the sentence instructions on their own without numbers or indented commands.